### PR TITLE
refactor(components): refactor tag

### PR DESCRIPTION
**Pull request by sxzz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/tag/src/tag.vue [hunk 1]
```diff
@@ -31,67 +31,41 @@
...
+    const tagSize = computed(() => props.size || ELEMENT.size)
     const classes = computed(() => {
       const { type, hit, effect } = props
       return [
         'el-tag',
         type ? `el-tag--${type}` : '',
         tagSize.value ? `el-tag--${tagSize.value}` : '',
         effect ? `el-tag--${effect}` : '',
-        hit && 'is-hit',
+        hit ? 'is-hit' : '',
```
Comment (JeremyWuuuuu) [inline packages/components/tag/src/tag.vue:54]: I think `hit && 'is-hit'` has no needs to change.

Comment (JeremyWuuuuu) [reply packages/components/tag/src/tag.vue:54]: `Vue` class combination will remove falsy(or non text type) class for us so that we don't need to do this.

**Review by jw-foss — APPROVED** [review]
Left a nitpick for you to consider, not blocking, approving as-is.

**Comment by sxzz** [comment]
Done.

**Pull request merged**